package.json

```diff
@@ -11,14 +11,20 @@
   },
   "dependencies": {
     "@next/font": "13.1.2",
+    "@svgr/cli": "^6.5.1",
```

Comment:
What are you using this for?

Reply:
This answer is yet unanswered, if you don't need this package, you might want to remove it.